### PR TITLE
No sudo for travis

### PR DESCRIPTION
**Pull request by magopian**

### CODE REVIEW

**Comment by Natim** [comment]
I think we can also remove mentions of python in ``.travis.yml``

**Comment by Natim** [comment]
```
- language:
-   - node_js
-   - python
- 
- python:
-     - "2.7"
```

```
+ language: node_js
```

File: loadtests/Makefile [hunk 1]
```diff
@@ -12,7 +12,7 @@ INSTALL = ARCHFLAGS=$(ARCHFLAGS) ./venv/bin/pip install
 .env.install:
 	virtualenv ./venv
 	$(INSTALL) pexpect >> loadtest.log && \
-	$(INSTALL) configparser==3.3.0r2 >> loadtest.log && \
+	$(INSTALL) configparser==3.3.0post2 >> loadtest.log && \
```
Comment (magopian) [inline loadtests/Makefile]: It seems they have a bit of a naming issue. The latest version is 3.5.0b2, which is a beta, not sure if you want to use that instead?

Comment (Natim) [inline loadtests/Makefile]: I recall that the 3.5.0b2 is not compatible with 3.3.0r2 nor 3.3.0post2 I am afraid.

Comment (magopian) [inline loadtests/Makefile]: 3.3.0post2 is working at least according to travis ;) I'm doing another run without the `python` language, just to see if it succeeds to, as you advised ;)

**Comment by Natim** [comment]
Thanks :+1:

**Pull request merged**